setup.py

```diff
@@ -371,7 +371,7 @@ def check_file_at_path(path2file):
 
 packages = find_packages(include=["numba", "numba.*"])
 
-build_requires = ['numpy >={}'.format(min_numpy_build_version)]
+build_requires = ['numpy >={}'.format(min_numpy_build_version), 'unittest-xml-reporting']
```

Comment:
The test failure may be caused by some broken packages with xml. Try removing 'unittest-xml-reporting' on this line so the py3.10 build is not trying to install it from `pip`. 

Later, we will need to compare the xml packages for py3.9 vs py3.10

Reply:
Thanks! That worked